app/scripts/views/settings/devices.js

```diff
@@ -75,7 +50,13 @@ define(function (require, exports, module) {
 
     _formatDevicesList: function (devices) {
       return _.map(devices, function (device) {
-        device.lastAccessTime = $.timeago(Number(device.lastAccessTime));
+        if (device.lastAccessTimeFormatted) {
+          device.lastAccessTime = Strings.interpolate(
+            t('Last active: %(translatedTimeAgo)s'), { translatedTimeAgo: device.lastAccessTimeFormatted });
```

Comment:
Is it possible to do something like how it was done in the mailer for location? The word `estimated` is appended to the city and is translated.

https://github.com/mozilla/fxa-auth-mailer/blob/master/mailer.js#L159